src/Agent/NewRelic/Agent/Core/Aggregators/MetricAggregator.cs

```diff
@@ -44,6 +45,19 @@ public MetricAggregator(IDataTransportService dataTransportService, IMetricBuild
             _metricStatsCollectionQueue = CreateMetricStatsCollectionQueue();
         }
 
+        protected override TimeSpan HarvestCycle
+        {
+            get
+            {
+                if (_configuration.MetricsHarvestCycle.HasValue)
+                {
+                    return _configuration.MetricsHarvestCycle.Value;
+                }
+
+                return DefaultHarvestCycle;
```

Comment:
Should we move the default value into DefaultConfiguration so that the logic for the harvest cycle is in just one location?